src/lib.rs

```diff
@@ -82,6 +82,10 @@ pub struct Frame {
     ///
     /// Only set for files that use simple blocks.
     pub is_discardable: Option<bool>,
+    /// Set when a duration is associated with a frame. (a BlockDuration):  if exist
```

Comment:
The documentation looks awkwards. Can you please simplify it? For example, you don't need to mention subtitles and also "BlockDuration" is an implementation detail, which doesn't need to be exposed to the documentation.